pkg/webhooks/group/group.go

```diff
@@ -114,7 +115,7 @@ func (s *GroupWebhook) authorized(request admissionctl.Request) admissionctl.Res
 			}
 		}
 		log.Info("Denying access", "request", request.AdmissionRequest)
-		ret = admissionctl.Denied("May not access protected group")
+		ret = admissionctl.Denied(fmt.Sprintf("Prevented from accessing Red Hat managed groups. Customer user groups may NOT match this regular expression: %s", protectedGroups))
```

Comment:
We don't need to include the regex, just that they're not permitted.

Reply:
The suggestion was to include the list of groups. However, that has the drawback where it's going to be hard to update as OSD matures. This is a proposal to meet halfway.